### DISCUSSION TITLE
Connect BudiBase DB to Adalo via API

### DISCUSSION THREAD

**Discussion by ytrewq17** [Budibase/budibase#10890] [Help]
Hi everyone,

I have to connect a Budibase DB to [Adalo](https://help.adalo.com/integrations/external-collections-with-apis) via API.

As you can see on the attachments, Adalo asks to setup a "Results Key" for the "Get All Records Endpoint".

The BudiBase DB data structure is under the "data" key but if I set "data" as the Results Key in my setup it doesn’t work. 

Could anyone help me understand how to get it work?

Thanks 🙏 

<img width="673" alt="2" src="https://github.com/Budibase/budibase/assets/135861080/62e7931f-17c3-4f25-843b-764b2e2e3286">
<img width="667" alt="1" src="https://github.com/Budibase/budibase/assets/135861080/115c070b-a8c1-45d0-ad60-06ca3104d10d">
<img width="674" alt="3" src="https://github.com/Budibase/budibase/assets/135861080/3c3fd0d5-23c3-4702-9976-55ee92f86e57">

**Comment by melohagan** [comment]
Hey @ytrewq17 

I've never used Adalo, but my guess here is that the *Get ALL* method expects an array of objects, whereas the tables endpoint returns an object for the table schema.

Perhaps you could try the *Get ONE* method...
